spec/index.yaml

```diff
@@ -1209,11 +1209,7 @@ paths:
         - Polaris_Management_Service_OAuth2: []
     delete:
       operationId: deleteCatalog
-      description: >-
-        Delete an existing catalog. This is a cascading operation that deletes
-        all metadata, including principals, roles and grants. If the catalog is
-        an internal catalog, all tables and namespaces are dropped without
-        purge.
+      description: Delete an existing catalog. The catalog must be empty.
```

Comment:
This PR itself LGTM, but I wonder how this spec file is related to the Iceberg REST Catalog spec... I see some paths match the Iceberg REST paths... I'd assume this whole file is meant to be 1:1 with the Iceberg spec... WDYT?

Reply:
Yes, it should be exact the same for anything in Iceberg REST spec. I think this file also provides specs that only in Polaris, like catalog endpoint.

Reply:
I also opened a thread for this in Zulip Chat.

Reply:
This spec file is specific to the Polaris management APIs - they aren't part of the Iceberg catalog spec, nor should they be. There shouldn't be any correspondence to catalog APIs. The actual Iceberg catalog rest spec that we implement lives here: https://github.com/apache/polaris/blob/main/spec/rest-catalog-open-api.yaml

Reply:
We got three yaml file in the directory
1. polaris-management-service.yml
2. rest-catalog-open-api.yaml
3. index.yaml

The index.yaml is actually a generated file combined by 1 and 2. I think that's why the confusion came from. Should we remove the generated file in the repo?
```
docker run -p 8080:80 -v ${PWD}:/spec --rm docker.io/redocly/cli join spec/docs.yaml spec/polaris-management-service.yml spec/rest-catalog-open-api.yaml -o spec/index.yaml --prefix-components-with-info-prop title
```